### PR TITLE
#422 : add ion/ion beam instability as functional test

### PR DESCRIPTION
**Pull request by rochSmets**
We should better rename this test ion/ion beam instability (following the Gary terminology) instead of 2 streams...

### CODE REVIEW

**Review by nicolasaunai — APPROVED** [review]

File: tests/functional/ionIonBeam/ion_ion_beam1d.py [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+
+####################################################################
+#                      post processing
+####################################################################
+
+def getMode(t, m):
+    return np.absolute(np.fft.fft(t)[m])
+
+def croaCroa(x, a, b):
```
[the remark this answers is not in the document]
Comment (rochSmets) [reply tests/functional/ionIonBeam/ion_ion_beam1d.py:113]: I add a comment to clarify that it calculate the FFT, and then return the mode m. hence, m=1 is the first harmonic that is with wavelength equal the length of the box

[the remark this answers is not in the document]
Comment (nicolasaunai) [reply tests/functional/ionIonBeam/ion_ion_beam1d.py:114]: I meant, what's croaCroa ?

[the remark this answers is not in the document]
Comment (nicolasaunai) [reply tests/functional/ionIonBeam/ion_ion_beam1d.py:114]: image


??

**Comment by nicolasaunai** [comment]
depends on #557

File: tests/functional/ionIonBeam/ion_ion_beam1d.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (rochSmets) [reply tests/functional/ionIonBeam/ion_ion_beam1d.py]: done

**Comment by rochSmets** [comment]
the test should be ok, rebase on master, with a single commit.

**Pull request merged**